src/features/VisualNovel/components/RelationCard/RelationCard.tsx

```diff
@@ -41,7 +46,18 @@ const RelationCardComponent: VFC<RelationCardProps> = ({
     bg="gray.100"
     gridTemplateRows="100%"
   >
-    <Image src={image ?? undefined} fallbackSrc={vnPosterPlaceholder} borderRadius="lg" h="full" w="full" objectFit="cover" />
+    <SafeImage
+      src={image ?? undefined}
```

Comment:
I think `SafeImage` should accept `string | null` for our convenience, because in our models images are `string | null`.